cloud_function/deploy.sh

```diff
@@ -6,10 +6,10 @@ TRIGGER_BUCKET=${2}
 TRIGGER_EVENT=${3:-"google.storage.object.finalize"}
 REGION=${4:-"us-central1"}
 
-if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev" ]; then
+if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev-storage" ]; then
   # This service account must have access to the WFL API
   SA_EMAIL="aou-cloud-fn-non-prod@broad-gotc-dev.iam.gserviceaccount.com"
```

Comment:
```suggestion
  SA_EMAIL="aou-cloud-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com"
```
Email got updated via https://github.com/broadinstitute/gotc-deploy/pull/115